src/components/presentational/ProgressBar/ProgressBar.stories.css

```diff
@@ -0,0 +1,3 @@
+.sb-story {
```

Comment:
What does this CSS do?

Reply:
Trying to get some padding in the storybook component. I replaced this with an extra div in the Template
```
const Template: StoryFn<typeof ProgressBar> = (args: ProgressBarProps) => <Layout colorScheme="auto"><div style={{padding: "35px 0"}}><ProgressBar {...args} /></div></Layout>;
```